.github/workflows/test-package-downloads-action-linux.yml

```diff
@@ -39,6 +39,11 @@ on:
         required: true
         type: string
         description: The nox version to install
+      python-version:
```

Comment:
Seeing changes in this file makes me think we should do a test staging run.  Thoughts?

Reply:
Yes, but lets merge this one first?